judgels-legacy/sandalphon/app/org/iatoki/judgels/sandalphon/controllers/api/client/v2/ClientProblemAPIControllerV2.java

```diff
@@ -134,13 +137,18 @@ public Result getBundleProblemWorksheet(String problemJid) {
             List<BundleItem> items = bundleItemService.getBundleItemsInProblemWithClone(problemJid, null);
             List<Item> itemsWithConfig = new ArrayList<>();
             for (BundleItem item : items) {
-                String itemConfig = bundleItemService.getItemConfInProblemWithCloneByJid(
+                String itemConfigString = bundleItemService.getItemConfInProblemWithCloneByJid(
                         problemJid, null, item.getJid(), language);
+                ItemType type = ItemType.valueOf(item.getType().name());
+
+                ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Please put this as a static final field in this class, so that it is only instantiated once

Reply:
Fixed to use injected objectMapper

Reply:
```
private static final ObjectMapper mapper = new ObjectMapper().registerModule(new Jdk8Module());
```